runtime/lua/vim/lsp.lua

```diff
@@ -2090,13 +2090,14 @@ api.nvim_create_autocmd('VimLeavePre', {
 ---@param bufnr (integer) Buffer handle, or 0 for current.
 ---@param method (string) LSP method name
 ---@param params table|nil Parameters to send to the server
----@param handler lsp-handler|nil See |lsp-handler|
```

Comment:
Can we keep the more concrete type here? 

There is a `@class` or `@alias` definition for lsp-handler

Reply:
right found it.